hello-world.js

```diff
@@ -1,3 +1,7 @@
+import someSecret from "secrets:///secretObject/subSecret";
```

Comment:
The hello-world app is getting complex and fails without dependencies configured. Maybe this could move to a new app in tests.